modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -140,6 +141,7 @@ public AbstractTypeScriptClientCodegen() {
         instantiationTypes.put("array", "Array");
 
         typeMapping = new HashMap<String, String>();
+        typeMapping.put("Set", "Set");
```

Comment:
@Persi You also need the lowercase mapping here: `typeMapping.put("set", "Set");`

Reply:
Thank you for your feedback, i've added your mentioned line.